### ISSUE TITLE
Possible to use without Hiredis?

### ISSUE THREAD

**Issue by pehrlich2** [kigster/simple-feed#33]
Hello!

Doing first-time setup here.  Following the given installation instructions on ruby 3.2.2 and either OSX or Heroku, I get the following error when starting the server:

```
LoadError: cannot load such file -- redis/connection/hiredis
```

Any tips for how to troubleshoot this? Is it possible to opt out of Hiredis, even temporarily?  Thanks!

**Comment by pehrlich2** [comment]
Looks like this may be the root cause - perhaps there are some changes on the side of simple-feed for compatibility here https://github.com/redis/redis-rb/issues/1178

**Comment by kigster** [comment]
@pehrlich2 **thank you so much for reporting this.** I would like to clarify your request:

- Would you rather see `hiredis` be optional or 
- do I simply need to change the require to match the recent changes in the `hiredis` gem?

The reason for `hiredis` to begin with is that it provides a much faster communications with the redis server, where the most high throughput functions are written in plain C.

Let me know!

**Comment by pehrlich2** [comment]
Hi!

I think it would be fine to simply update the require statement - as long
as that doesn’t cause new problems. Have been using it that way and no
trouble so far.

Cheers

**Comment by kigster** [comment]
Any chance you might be open to submitting a PR with your solution? 

It be great if hiredis was optional, but the README explained how to enable it for high throughput.

**Comment by kigster** [comment]
Is this still an issue that need solving?

**Comment by bkmaibach** [comment]
Yes, please solve this!

**Comment by bkmaibach** [comment]
> Is this still an issue that need solving?

Yes please!

**Comment by bkmaibach** [comment]
If you are attempting to use this gem without hiredis (standary ruby driver only) in rails with Redis >5 you can do the following:

Add the following line to boot.rb:
$LOAD_PATH.unshift(File.expand_path('../../lib/overrides', __FILE__))

And then add an empty file at:
lib/overrides/redis/connection/hiredis.rb

**Comment by kigster** [comment]
Thanks. I'll release a fix for this shortly

**Comment by kigster** [comment]
Fixed in version 3.2.0, by https://github.com/kigster/simple-feed/pull/38 and https://github.com/kigster/simple-feed/pull/35
